src/lib/device/device.h

```diff
@@ -149,6 +149,7 @@ class DeviceManager {
 
     bool is_first_time_boot() const { return m_first_time_boot; }
     void format_devices();
+    void commit_formatting();
```

Comment:
@yamingk The files device.h, device_manager.cpp, hs_super_blk.h and homestore.cpp are relevant changes regarding taking cp on first_time_boot related changes. Please review these.